libzerocash/utils/util.cpp

```diff
@@ -182,7 +182,7 @@ void concatenateVectors(const std::vector<unsigned char>& A, const std::vector<u
     result.insert(result.end(), C.begin(), C.end());
 }
 
-void sha256(unsigned char* input, unsigned char* hash, int len) {
+void sha256(const unsigned char* input, unsigned char* hash, int len) {
```

Comment:
[dnb] Great! Let's start ratcheting down compiler checks whenever we can correctly introduce tighter static constraints / types.